lib/encoder.rb

```diff
@@ -0,0 +1,17 @@
+require './lib/crypto_tool'
+
+class Encoder < CryptoTool
+
+  def random_key
+    key = "00000"
+    key.split('').map {|char| rand(10).to_s}.join
```

Comment:
changed this to randomize each individual digit. did this because I'm not 100% sure `rand` will always return at least 6 digits after the decimal point (average seems to be about 17 digits; the lowest I got was 8 digits, so I imagine it's possible but extremely rare)